### PR TITLE
Release/fix fork and registration wikis [OSF-6095]

### PR DESCRIPTION
**Pull request by samanehsan**
## Purpose
When a node is forked or registered, the `wiki_pages_current` and `wiki_pages_versions` lists are copied over, but not the `NodeWikiPage` objects themselves. As a result, a fork or registration can include a wiki version where its `node` field is the original project. Because the wiki node id does not match the fork/registration id, the API throws an error when a user tries to comment on one of those wiki pages.

## Changes
When a project is forked or registered, create a copy of each wiki page in `wiki_pages_versions` and change the node to the fork or registration instead.

## Side effects
More NodeWikiPages!

Requires migration:

```
python -m scripts.clone_wiki_pages
```

## Ticket
https://openscience.atlassian.net/browse/OSF-6095

### CODE REVIEW

File: scripts/clone_wiki_pages.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+def clone_wiki_pages(nodes):
+    for i, node in enumerate(nodes):
+        if node.wiki_pages_versions:
+            cloned_wiki_pages = {}
+            for key in node.wiki_pages_versions:
+                cloned_wiki_pages[key] = []
+                for wiki_id in node.wiki_pages_versions[key]:
+                    node_wiki = NodeWikiPage.load(wiki_id)
+                    if node_wiki.node._id != node._id:
+                        clone = node_wiki.clone()
```
Comment (sloria) [inline scripts/clone_wiki_pages.py]: As mentioned on FD: can you refactor this into a method of NodeWikiPage to DRY up the code?

File: scripts/clone_wiki_pages.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                        clone.node = node
+                        clone.user = node_wiki.user
```
Comment (sloria) [inline scripts/clone_wiki_pages.py]: Is this line necessary?

Comment (samanehsan) [inline scripts/clone_wiki_pages.py]: Yes, `user` gets set to `None` when the `NodeWikiPage` is cloned so this line fixes that.

**Comment by sloria** [comment]
Fixed up the migration in https://github.com/CenterForOpenScience/osf.io/commit/c91d2649a9838e6beec022dab15710f4bdfd65b6 and merged this into release/0.68.0. 👍 🐙

**Pull request closed without merging**